### ISSUE TITLE
Allow Tasker actions to change any setting inside Muzei

### ISSUE THREAD

**Issue by dariusz-goc** [romannurik/muzei#547]
Change blur amount, dimming, artwork source etc. to be changed from Tasker. Smoothly animated just like current double tap to unblur.

**Comment by xd1936** [comment]
It would be a dream come true if Muzei could set Tasker (or [Automate](https://llamalab.com/automate/)) itself as an Art Source, and Tasker could then feed URLs, Titles, Bylines, etc to Muzei for recipe-based images. Maybe #376 could be revisited.

I know [AutoTools](https://joaoapps.com/autotools/) has the ability to be set as a Muzei Art Source to similar effect. It would be so cool if that was built-in.

**Comment by dariusz-goc** [comment]
That's already possible with plugins. I'm for controlling blur and fade through tasker.

**Comment by ianhanniballake** [comment]
Once #368 is done, it might only be possible to enable / disable individual effects rather than have precise control (since every effect may have a completely different set of controls). Still seems like a useful addition though. Changing the source via Tasker is already covered in #417, so this feature request will cover adjusting effects via Tasker.

**Comment by dariusz-goc** [comment]
From what I can see in the Muzei settings all 3 effects (blur, dim, grey) are controlled by sliders. Only "Apply blur on lockscreen" is a switch. I would really like those to be controllable from Tasker. Not to be confused with "Lenses" #368 that add another effects on top of what Muzei is generating, eg. rain drops or color transformations etc. I don't think those should completely replace built-in settings as they are today, rather be added on top.

**Comment by ianhanniballake** [comment]
That's not what #368 would do - **every** effect, from the built in effects (blur dim, grey) to third party effects, would follow the same system - binary on/off plus optional, effect specific settings (ala the current sliders). Therefore the common denominator that Muzei could provide to Tasker would only be the on/off.

Certainly will consider Tasker integration as one offs for each built in effect though.

(Also note that the 'Apply blur on lockscreen' has been completely replaced in [Muzei 3.0 Alpha 2](https://plus.google.com/+IanLake/posts/DiKA5FbZ443) with completely separate controls for home screen and lock screen)

**Comment by dariusz-goc** [comment]
I see, so you want to bring everything under the same umbrella. Are all of the effects gonna be combinable? That could give unexpected results with third party effects, that's why I was under the assumption that you'll keep built-in effects separate and only allow one third party effect at the time. But that's all up to you of course. I'm happy that you'll consider adding more individual controls to Tasker, that's all I could ask for. I just hope that separate set of controls for lockscreen won't interfere with the Tasker implementation.

**Comment by xcom923** [comment]
is there any update on this? being able to adjust the saturation or blur depending on the time of day would be nice. I have ideas that would be cool to execute in tasker.
